.expeditor/buildkite/cypress.sh

```diff
@@ -17,6 +18,21 @@ do
   for version in ${versions[*]}
   do
     if jq -enr --arg version "$version" --arg fqdn "$instance" --argjson data "$data" '$data[] | select(.fqdn == $fqdn) | .tags | any(. == $version)'; then
+      if [[ "$version" == "v1" ]]; then
+        if ! token=$(chef-automate admin-token); then
+          echo "Non-zero exit code, output:"
+          echo "$token"
+          return 1
+        fi
+      else 
+        if ! token=$(chef-automate iam token create $timestamp-tok --admin); then
+          echo "Non-zero exit code, output:"
+          echo "$token"
+          return 1
+        fi
+      fi
+      
+      export CYPRESS_ADMIN_TOKEN=$token
```

Comment:
all the changes in this commit are to ensure an admin token is always available to the Cypress test runner.

Reply:
yaaaay